src/functions/contactsMenu/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+type ContactsMenuCallback = () => void
+
+interface ContactsMenuHookProps {
+	id: string;
+	callback: ContactsMenuCallback;
+}
+
+declare global {
+	interface Window {
+		_vue_contacts_menu_hooks: Record<string, ContactsMenuHookProps>;
```

Comment:
naming should be consistent with other so better `_nc_...`

Reply:
Done